webview-ui/src/components/chat/ChatRow.tsx

```diff
@@ -1290,58 +1359,10 @@ export const ChatRowContent = memo(
 					case "api_req_started":
 						return (
 							<>
-								<div
-									onClick={handleToggle}
-									style={{
-										...headerStyle,
-										marginBottom:
-											(cost == null && apiRequestFailedMessage) || apiReqStreamingFailedMessage ? 10 : 0,
-										justifyContent: "space-between",
-										cursor: "pointer",
-										userSelect: "none",
-										WebkitUserSelect: "none",
-										MozUserSelect: "none",
-										msUserSelect: "none",
-									}}>
-									<div
-										style={{
-											display: "flex",
-											alignItems: "center",
-											gap: "10px",
-										}}>
+								{title && (
+									<div style={headerStyle}>
 										{icon}
 										{title}
-										{/* Need to render this every time since it affects height of row by 2px */}
-										<VSCodeBadge
-											className={cn("text-sm", {
-												"opacity-100": cost != null && cost > 0,
-												"opacity-0": cost == null || cost <= 0,
-											})}
-											style={{
-												opacity: cost != null && cost > 0 ? 1 : 0,
-											}}>
-											{cost != null && Number(cost || 0) > 0 ? `$${Number(cost || 0).toFixed(4)}` : ""}
-										</VSCodeBadge>
-									</div>
-									<span className={`codicon codicon-chevron-${isExpanded ? "up" : "down"}`}></span>
-								</div>
-								{((cost == null && apiRequestFailedMessage) || apiReqStreamingFailedMessage) && (
-									<ErrorRow
```

Comment:
<img width="387" height="126" alt="Image" src="https://github.com/user-attachments/assets/b097c423-bb89-40de-8f4d-36723935dff6" />

Removing API request toggle might not be the best idea because although its rarely used it can potentially be helpful and it doesn't take any extra space. 

Are we sure we wanna remove it?